src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,96 @@
+.product-card {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px auto 40px;
+  }
+
+  &__infoblock {
+    display: flex;
+    flex-direction: column;
+    padding: 0 16px 16px;
```

Comment:
Let's remove infoblock wrapper, and add general paddings to product-card wrapper